scala/src/com/google/idea/blaze/scala/run/producers/ScalaTestContextProvider.java

```diff
@@ -76,27 +104,31 @@ private static ScClass getTestClass(ConfigurationContext context) {
       // handled by a different producer
       return null;
     }
-    return getTestClass(location);
+    ScalaTestConfigurationProducer scalaTestConfigurationProducer = new ScalaTestConfigurationProducer();
```

Comment:
We could use ScalaTestConfigurationProducer.instance() here

Reply:
:+1: Addressed